src/main/java/seedu/address/model/timetable/TimeBlock.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.model.timetable;
+
+import org.joda.time.Hours;
```

Comment:
Be sure to acknowledge any dependencies in the DG!

Reply:
Thank you for the reminder.

Reply:
Reminder to self: README also need acknowledge!